test/models/translations_test.rb

```diff
@@ -17,4 +17,50 @@ class TranslationsTest < ActiveSupport::TestCase
                    Translations.instance.for_js[l].keys.sort
     end
   end
+
+  # What tags & attributes are allowed?
+  ACCEPTABLE_TAGS = %w[h1 a strong em i b small tt ol ul li br p span].freeze
+  # Class can cause trouble, but we need it for glyphicons, etc.
+  ACCEPTABLE_ATTRS = %w[href name class target].freeze
+
+  def sanitize_html(x)
```

Comment:
Rename fine, but "input" is misleading (none of these are user input). Maybe "text"?